src/v/net/conn_quota.h

```diff
@@ -17,6 +17,8 @@
 #include "utils/mutex.h"
 #include "vassert.h"
 
+#include <seastar/core/smp.hh>
```

Comment:
`oncore.h` does not require `smp.hh` (same in the .cc file)

Reply:
Done